docs/sphinx/methods/dsrg.rst

```diff
@@ -16,10 +16,27 @@ Driven Similarity Renormalization Group
   Depending on the features used, the user is encouraged to cite the corresponding articles listed :ref:`here <dsrg_ref>`.
 
 .. caution::
-  The current implementation does not employ spin-adapted equations and it does not work for even spin multiplicities (doublet, quartets, etc.).
-  For odd multiplicities, we assume low-spin configurations (by default, no need to specify in the input file).
-  For those in desperate need to perform computations on doublets, an alternative way is to add a hydrogen atom far away from the system and perform a singlet computation.
-  Spin adaptation is on the TODO list.
+  The examples used in this manual are written based on the spin-integrated code.
+  To make the spin-integrated code work properly for molecules with even multiplicities,
+  the user should specify the following keyword:
+  ::
+
+     spin_avg_density    true       # use spin-summed reduced density matrices
```

Comment:
Is this obsolete now? If it is, it might still be a good idea to mention it somewhere, either here or in a section on active space solvers.

Reply:
BTW, how do you feel about moving our manual to jupyter?

Reply:
No, it is not obsolete. It shows the user should specify this keyword to run jobs with even multiplicities using the spin integrated code. I will emphasize this using bold font.

I am OK to move them to Jupyter but not sure what the benefit is. I assume they will still be markdown in Jupyter.